plugins/salesforce-commerce-api-plugin/src/api.ts

```diff
@@ -90,6 +90,17 @@ export class Api {
     }
   }
 
+  async validateConfig(){
+    try{
+       const response = await this.request(`validate-config`);
+       if(response.errors){
+        throw response.errors[0]?.title;
+       }
+    }catch(e){
+      throw e;
```

Comment:
if we are just throwing error, we can avoid the try catch. Check for `response.errors[0]?.title` at the location where we are catching the error

Reply:
right! removed the try catch